### DISCUSSION TITLE
Master Unlocker 1.40.0

### DISCUSSION THREAD

**Discussion by jhowkNx** [jhowkNx/Master-Unlocker#56] [Announcements]
## Report bugs or suggestions.

**Comment by liquidwater01** [comment]
Special cargo instant finish does not work when the mission involves going to pick up satchel charges then swimming to blow open containers under water. Upon exiting the office and after the phone call, activate Instant Finish Special Cargo (Buy) and the mission fails saying the cargo was not delivered. Does instant finish only work for certain missions?

Instant sell special cargo 1 crate 6m bugs with Transaction Pending. I noticed you added a message saying switch sessions, but no money appears after switching sessions. Are you saying switch sessions and sell until the transaction works?

**Reply by jhowkNx** [reply]
You should get paid before the pending transaction message, you won't get paid before then. When this appears, it will take a LONG time for you to get paid again, so it is recommended to switch sessions.
Regarding Instant Finish, I don't have much to say as it depends on many aspects of the game.
